phoneApp/js/app.js

```diff
@@ -0,0 +1,183 @@
+class App {
+	constructor(){
+		this.state = {	//равен ссылке, которая ведет на объект
+			people: [
+					  {
+				        name: 'Иван',
+				        lastName: 'Петров',
+				        email: 'IvanPetrov@ec.ua'
+				      },
+				        {
+				        name: 'Сергей',
+				        lastName: 'Сергеев',
+				        email: 'SergeiSergeev@ec.ua'
+				      },
+				        {
+				        name: 'Иван',
+				        lastName: 'Иванов',
+				        email: 'IvanIvanov@ec.ua'
+				      },
+				        {
+				        name: 'Александр',
+				        lastName: 'Александров',
+				        email: 'AlexAlex@ec.ua'
+				      },
+				        {
+				        name: 'Алекс',
+				        lastName: 'Смирнов',
+				        email: 'AlexSmirnov@ec.ua'
+				      },
+				        {
+				        name: 'Сергей',
+				        lastName: 'Волков',
+				        email: 'VolkovSergey@ec.ua'
+				      },
+				        {
+				        name: 'Мария',
+				        lastName: 'Шарапова',
+				        email: 'MariyaSharapova@ec.ua'
+				      },
+				        {
+				        name: 'Александр',
+				        lastName: 'Винник',
+				        email: 'AlexVinnik@ec.ua'
+				      },
+				        {
+				        name: 'Дарий',
+				        lastName: 'Смирнов',
+				        email: 'DariySmirnov@ec.ua'
+				      },
+				        {
+				        name: 'Елена',
+				        lastName: 'Лещенко',
+				        email: 'ElenaLeshenko@ec.ua'
+				      },
+				        {
+				        name: 'Ольга',
+				        lastName: 'Новикова',
+				        email: 'OlgaNovikova@ec.ua'
+				      },
+				        {
+				        name: 'Наталья',
+				        lastName: 'Шемякина',
+				        email: 'ShemyakinaN@ec.ua'
+				      },
+				        {
+				        name: 'Анна',
+				        lastName: 'Донцова',
+				        email: 'AnnaDontsova@ec.ua'
+				      },
+				        {
+				        name: 'Влад',
+				        lastName: 'Яма',
+				        email: 'VladYama@ec.ua'
+				      },
+				        {
+				        name: 'Кира',
+				        lastName: 'Воробьева',
+				        email: 'Kira1990@ec.ua'
+				      },
+				        {
+				        name: 'Виктор',
+				        lastName: 'Кривенко',
+				        email: 'ViktorKriv@ec.ua'
+				      }
+				    ],
+			activePage: 'contacts'
+		};
+
+        this.pages = {
+            contacts: new ContactsPage(this.state),	// тут передали ссылку на this.state
+			adduser: new AddUser(this.state),
+            keypad: new KeypadPage(this.state),
+            editcontact: new EditContact(this.state),
+            user: new User(this.state)
+        };
+
+        this.initializeRouter();
+        this.switchRouter();
+
+    }
+
+    initializeRouter () {
```

Comment:
pretty sure you can create an additional file and call it "Router.js" - and move all specific to navigation logic in.

Then initialize as other pages.

The main goal what we trying to solve the minimal splitting by responsibilities have you feel it?